.github/workflows/ci-aarchxx.yml

```diff
@@ -209,6 +209,7 @@ jobs:
         cd /tmp
         wget https://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
         unzip -q android-ndk-r10e-linux-x86_64.zip
+        export ANDROID_NDK_ROOT=/tmp/android-ndk-r10e
```

Comment:
Please fix the package build one as well (unfortunately these command sequences are just duplicated; maybe we could make little shell scripts, or maybe there's an include or function mechanism in .yml files?  not sure):
```
$ git grep android-ndk-r10e-linux-x86_64
.github/workflows/ci-aarchxx.yml:        wget https://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
.github/workflows/ci-aarchxx.yml:        unzip -q android-ndk-r10e-linux-x86_64.zip
.github/workflows/ci-package.yml:        wget https://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
.github/workflows/ci-package.yml:        unzip -q android-ndk-r10e-linux-x86_64.zip
api/docs/building.dox:wget https://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
api/docs/building.dox:unzip android-ndk-r10e-linux-x86_64.zip
```
Probably don't need to update the building.dox one.

Reply:
(To clarify: I'm not requesting that you solve the command duplication: if we did find a way to solve it, we would do a separate PR, and it's not high priority right now.)

Reply:
Look at this [answer](https://stackoverflow.com/a/15437697). It said YML files don't support the include mechanism.
And if I want to split this section into an script, which directory do you want to put it? make/.github, .github/scripts or tools/ci-scripts?

Reply:
Got it. Done.